composer.json

```diff
@@ -16,6 +16,9 @@
         "php": "^5.3 || ^7.0"
     },
     "require-dev": {
+        "doctrine/dbal": "^2.2",
+        "doctrine/mongodb-odm": "^1.0",
```

Comment:
You have to remove this or test on PHP7 will fail.

Don't worry for test, it will be required here: https://travis-ci.org/sonata-project/exporter/jobs/134259161